install.sh

```diff
@@ -33,9 +45,7 @@ readonly RELEASE_ARCH="amd64"
 
 # Finding the releases to download
 readonly GITHUB_OWNER_REPO='nats-io/nsc'
-readonly GITHUB_API_NSC="https://api.github.com/repos/$GITHUB_OWNER_REPO"
 readonly HTTP_USER_AGENT='nsc_install/0.1 (@nats-io)'
-readonly GITHUB_HTTP_ACCEPT='Accept: application/vnd.github.v3+json, application/json'
 
 # Where to install to, relative to home-dir
 readonly NSC_RELATIVE_BIN_DIR='.nsccli/bin'
```

Comment:
This always been the case?

Reply:
Has what always been the case?  The logic was there for talking to GitHub's API server and there's a bunch of features which only turn on with a correct `Accept:` header ... I forget the details, I ripped the string from my own `github_curl` wrapper.

When we're not talking to the API server, we don't need this constant.

Reply:
I meant this.. '.nsccli/bin'

Reply:
Oh.  Yes, that's where the Python script installs to.  I duplicated the functionality closely.

`git annotate` shows those lines as coming from commit 5c41b492 which is the commit which introduced the Python script, so "yes, always".

Reply:
??? if you run the python script it installs it in ~/.nsc, not ~/.nsccli

Reply:
Not quite sure how this came to be. - Indeed it does store it in .nsccli, really we should normalize this stuff, and store it in .nats